.github/workflows/ci.yml

```diff
@@ -33,6 +33,9 @@ jobs:
     - name: Install tools
       shell: bash
       run: "bundle install && gem install --no-document toys"
+    - name: Install Minitest 5.22.2
+      shell: bash
+      run: gem install minitest -v 5.22.2
```

Comment:
The minitest 5.22.2 issue is very bizarre. We'll need to dig more into it. Can you add TODO comment on it, noting that it's temporary to fix a weird CI issue and we need to circle back on it?

Reply:
Created an issue to track it as well.